crates/ruma-serde/src/raw.rs

```diff
@@ -23,6 +23,8 @@ use crate::cow::MyCowStr;
 /// to pass around events in a lossless way.
 ///
 /// ```ignore
+/// # // HACK: "ignore" the doctest here because `AnyRoomEvent` comes from ruma_events, which cannot
+/// # // be depended on without creating a cyclical dependency.
```

Comment:
I should have explained how I was thinking these could be changed to `no_run`. The way I was imagining here: Just define a hidden `struct AnyRoomEvent;` that derives `Deserialize` 😉